pkg/services/ngalert/models.go

```diff
@@ -0,0 +1,75 @@
+package ngalert
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/grafana/grafana/pkg/services/ngalert/eval"
+	"github.com/grafana/grafana/pkg/tsdb"
+)
+
+// AlertDefinition is the model for alert definitions in Alerting NG.
+type AlertDefinition struct {
+	Id        int64
+	OrgId     int64
+	Name      string
+	Condition string
+	Data      []tsdb.Query
```

Comment:
I think we may want to make this a backend.DataQuery as soon as we can. That has the timerange attached to the query, which we will want to get in ASAP on the frontend.